### ISSUE TITLE
failure with jbuilder partials

### ISSUE THREAD

**Issue by net1957** [camertron/arel-helpers#46]
For a long time, I'm using QueryBuilder to construct complicated sql queries.

I'tried to convert my jbuilder view to use partials

It failed because jbuilder call empty? and size methods on partial calls

With this monkey patch:
````
module ArelHelpers
  class QueryBuilder
    def_delegators :@query, :empty?, :size
  end
end
````

it's OK

Could you update ?
Regards

**Comment by camertron** [comment]
Hey @net1957, thanks for taking the time to file an issue. Given that we already forward a number of methods to `@query` I think it isn't a big deal to add a couple more. Can you create a pull request?

**Comment by net1957** [comment]
Done

**Comment by camertron** [comment]
Merged and published in v2.13.0.
